astropy/io/ascii/ui.py

```diff
@@ -192,7 +192,7 @@ def read(table, guess=None, **kwargs):
         Input table as a file name, file-like object, list of strings,
         single newline-separated string or pathlib.Path object .
     guess : bool
-        Try to guess the table format (default= ``True``)
+        Try to guess the table format (default=True)
```

Comment:
This is worse: the default is not in fact `True` but `None`, which means it gets set to `_GUESS` below, which for some obscure reason is not a config parameter, but something that can be set by `set_guess` -- a separate issue, but I think this description should state what currently happens...